tdi_python/tdicli.py

```diff
@@ -2147,15 +2139,33 @@ def unload_ipython_extension(ipython):
         ipython.input_transformers_cleanup.remove(tdi_input_transform)
     pass
 
+def destroy_tdi_python_session():
+    sts = self._driver.tdi_session_destroy(tdi_session)
+    if not sts == 0:
+        print("ERROR: destroying session")
+        self._driver.dev_config_deallocate(self.dev_config)
+        return -1
+    return 0
+
+def create_tdi_python_session():
```

Comment:
What will be the device here in a multi-device system?

Reply:
changed now